.gitignore

```diff
@@ -1 +1,4 @@
 /vendor/
+composer.lock
```

Comment:
Are we sure we want to ignore our `composer.lock`?

Reply:
As currently the only downloadable requirement is `phar-io/composer-distributor` I'd say we don't need the `composer.lock` file. We currently have complete control over what is installed. Should that change we can of course modify that.